src/main/java/racingcar/model/domain/RandomNumberBehavior.java

```diff
@@ -0,0 +1,29 @@
+package racingcar.model.domain;
+
+public class RandomNumberBehavior implements CarMoveBehavior {
+
+    private final static int MAX_RANDOM_RANGE = 10;
+    private final static int MIN_BOUND_STRAIGHT = 4;
+    private static RandomNumberBehavior randomNumberBehavior;
```

Comment:
불변이라면 final을 붙여보는 건 어떨까요?

```suggestion
    private static final RandomNumberBehavior randomNumberBehavior;
```